server/app/services/applicant/ApplicantService.java

```diff
@@ -1216,9 +1221,28 @@ private ApplicationPrograms relevantProgramsForApplicantInternal(
         .build();
   }
 
+  private ProgramDefinition getProgramDefinitionForDraftApplication(
```

Comment:
Java doc will be in the follow up PR. Without the code it's out of context and doesn't mean a whole lot..